gensim/models/coherencemodel.py

```diff
@@ -113,16 +113,18 @@ class CoherenceModel(interfaces.TransformationABC):
     def __init__(self, model=None, topics=None, texts=None, corpus=None, dictionary=None,
                  window_size=None, coherence='c_v', topn=10, processes=-1):
         """
-        Args:
```

Comment:
It gave an _unexpected section title_ and _expected indentation_ warning during `make clean html`. And it seemed like the[ reST Sphinx format](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#doctest-blocks) was being followed, according to which a keyword followed by a colon followed by some text is a [description list](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#definition-lists). And an underlined keyword is a [section title](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#sections). Also, text in a description list should be indented.